static/src/javascripts/components/curl/curl-domReady.js

```diff
@@ -1,4 +1,4 @@
-(function(){/*
+window.require || (function(){/*
```

Comment:
Yeah that is an option. I think it would be good to avoid a separate `app.js` if possible for simplicity, but also for things like caching etc.